File: 1264-maximum-number-of-words-you-can-type/1264-maximum-number-of-words-you-can-type.cs
public class Solution {
    public int CanBeTypedWords(string text, string brokenLetters) {

    string[] words = text.Split(" ");   
    int brokenCount = 0;

    foreach (var word in words) {
        foreach (char ch in brokenLetters) {
            if (word.Contains(ch)) {
                brokenCount++;
                break;
            }
        }
    }
    return words.Length - brokenCount;
        
    }
}

File: 1264-maximum-number-of-words-you-can-type/1264-maximum-number-of-words-you-can-type.js
/**
 * @param {string} text
 * @param {string} brokenLetters
 * @return {number}
 */
var canBeTypedWords = function(text, brokenLetters) {
    let count = 0;
    let words = text.split(" ");

    for (let word of words) {
        let canType = true;
        for (let ch of brokenLetters) {
            if (word.includes(ch)) {
                canType = false;
            }
        }
        if (canType) count++;
    }
    return count;
};